libs/dh/wholesale/feature-settlement-reports/src/lib/drawer/dh-settlement-report-drawer.component.html

```diff
@@ -97,12 +97,16 @@ <h4>{{ t("drawer.gridAreas") }}</h4>
           <span class="watt-chip-label">{{ tableSource.data.length }}</span>
         </vater-stack>
 
-        <watt-table
-          [dataSource]="tableSource"
-          [columns]="columns"
-          [hideColumnHeaders]="true"
-          [suppressRowHoverHighlight]="true"
-        />
+        @if (tableSource.data.length > 0) {
+          <watt-table
+            [dataSource]="tableSource"
+            [columns]="columns"
+            [hideColumnHeaders]="true"
+            [suppressRowHoverHighlight]="true"
+          />
+        } @else {
+          {{ t("noData") }}
```

Comment:
Question: Not our Empty component?

Reply:
Not here. Discussed it with Pernille and we agreed to go with it.